hack/golangci-lint.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2023 Red Hat, Inc.
```

Comment:
2024

Reply:
@EdDev @nirdothan Actually I don't think we need to specify the year anymore. [Link](https://github.com/kubevirt/kubevirt/blob/main/hack/boilerplate/boilerplate.go.txt)

Reply:
done